MVP Module/Source/Feed Presentation/FeedErrorViewModel.swift

```
@@ -0,0 +1,13 @@
+//
+//  FeedErrorViewModel.swift
+//  MVVMTests
+//
+//  Created by Salah Amassi on 06/03/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+struct FeedErrorViewModel {
+	let error: String
```

The `error` could be an optional `String?`, so the Presenter can decide when to show/hide the message.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `errorView.display(FeedErrorViewModel(error: .none))` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's often easier to manage state when all changes are centralized.